### PR TITLE
feat: move chat support behind paywall

### PR DESCRIPTION
**Pull request by YounixM**
https://www.loom.com/share/b97a8867798948c998b190f793833d6f?sid=b0fe14e7-08ff-4256-a127-6cbec534b811

### CODE REVIEW

File: frontend/src/AppRoutes/index.tsx [hunk 1]
```diff
@@ -62,10 +62,12 @@ function App(): JSX.Element {
 			allFlags.find((flag) => flag.name === FeatureKeys.ONBOARDING)?.active ||
 			false;
 
-		const isChatSupportEnabled =
+		const isChatSupportFFEnabled =
 			allFlags.find((flag) => flag.name === FeatureKeys.CHAT_SUPPORT)?.active ||
 			false;
 
+		const isChatSupportEnabled = isChatSupportFFEnabled;
```
Comment (SagarRajput-7) [inline frontend/src/AppRoutes/index.tsx:69]: NIT: Didn't get why this is done.

**Review by SagarRajput-7 — APPROVED** [review]
Please Add Screenshot, Code wise looks good to me.

File: frontend/src/AppRoutes/index.tsx (hunk 1 above)
Comment (YounixM) [reply frontend/src/AppRoutes/index.tsx:69]: Waiting for the API to get the required data.

File: frontend/src/container/AppLayout/index.tsx [hunk 2]
```diff
@@ -331,6 +385,58 @@ function AppLayout(props: AppLayoutProps): JSX.Element {
 					</Sentry.ErrorBoundary>
 				</div>
 			</Flex>
+
+			<div className="chat-support-gateway">
+				<Button
+					className="chat-support-gateway-btn"
+					onClick={(): void => setIsAddCreditCardModalOpen(true)}
```
Comment (vikrantgupta25) [inline frontend/src/container/AppLayout/index.tsx:392]: NIT : can we extract this out to a separate file ? we should keep the `AppLayout` lean.

Comment (YounixM) [reply frontend/src/container/AppLayout/index.tsx]: Moved ✅

File: frontend/src/AppRoutes/index.tsx (hunk 1 above)
Comment (YounixM) [reply frontend/src/AppRoutes/index.tsx]: Updated.

**Review by makeavish — APPROVED** [review]

**Pull request merged**